packages/ag-charts-community/src/chart/themes/chartTheme.ts

```diff
@@ -358,6 +359,13 @@ export class ChartTheme {
         return deepClone(themeInstance);
     }
 
+    protected static getDefaultColors(): { fills: { [key: string]: string }; strokes: { [key: string]: string } } {
```

Comment:
Nit: We're repeating this (return) typing in a bunch of places, could we extract a `type` or `interface` and re-use it?